disdrodb/l0/check_metadata.py

```diff
@@ -120,34 +122,25 @@ def identify_empty_metadata_keys(metadata_fpaths: list, keys: Union[str, list])
     return None
 
 
-def get_archive_metadata_key_value(disdrodb_dir, key, return_tuple=True):
```

Comment:
Please keep this (and the test) and move to `disdrodb.api.metadata`. It's really useful to extract information from the metadata archive !